src/main/groovy/com/morpheusdata/xen/XenserverCloudProvider.groovy

```diff
@@ -140,7 +141,7 @@ class XenserverCloudProvider implements CloudProvider {
 		options << new OptionType(
 				name: 'Enable Hypervisor Console',
 				code: 'xenServer-enable-hypervisor',
```

Comment:
match the code format established by other option types: `zoneType.xen.enableHypervisor`. Import existing is also not following the same pattern: `zoneType.xen.importExisting`

Reply:
@ddeyoung code updated as suggested.